### ISSUE TITLE
md-list md-item height issue

### ISSUE THREAD

**Issue by hodeyp** [angular/material#1682]
It seems that both md-list & md-item are inheriting display:inline and as such are set to 0px height. Content in the list overflows and displays but any element that is meant to be below the md-list does not display correctly.

**Comment by ThomasBurleson** [comment]
Please provide a CodePen or Plunkr that demonstrates this issue.

**Comment by Klaster1** [comment]
I've got the same issue too, here's the [CodePen](http://codepen.io/anon/pen/YPooWP). Press on the button to switch inherited `inline-block` and `block`.

**Comment by topherfangio** [comment]
Updating the Codepen to 0.10.1 appears to resolve the issue. @hodeyp @Klaster1 are you still seeing the issue?

**Comment by topherfangio** [comment]
Since we have not received feedback, I am going to go ahead and close this issue.

If you are still experiencing the problem with RC2 or greater, please post a Codepen/comment and we will reopen.

**Comment by joshterainsights** [comment]
Had a similar issue with md-tabs causing one arbitrary md-tab-content to have a height of 0px, but it was still `display: block` like all others and couldn't say what it was inheriting the height from.

**More importantly**, updating that to `1.0.0-rc3` fixed it.

So while it wasn't the *exact* same issue as OP's, I'd say it's close enough to confirm this ticket can remain closed.

### TL;DR: Confirming probably fixed by RC2
